test/e2e/tests/network/multi-rpc.spec.ts

```diff
@@ -173,7 +162,9 @@ describe('MultiRpc:', function (this: Suite) {
       },
 
       async ({ driver, mockedEndpoint }) => {
-        await unlockWallet(driver);
+        await loginWithoutBalanceValidation(driver);
```

Comment:
I wanted to propose a change around the method `loginWithBalanceValidation` which will allow us to use it in this kind of cases.
I think that if we add an optional argument for passing the expected balance, then we'll be able to pass that, if we enter the else condition, and that could then be used in cases like this test by doing
`await loginWithBalanceValidation({ driver,   expectedBalance, });`

The change in the function would be to add the args as an object (so we can better pass the params, and omit optional params with a readable form), as well as pass the expectedBalance to the else condition

```
interface LoginWithBalanceValidationParams {
  driver: Driver;
  ganacheServer?: Ganache;
  password?: string;
  expectedBalance?: string;
}

export const loginWithBalanceValidation = async ({ // args are now passed as an object
  driver,
  ganacheServer,
  password,
  expectedBalance, // added
}: LoginWithBalanceValidationParams) => {
  await loginWithoutBalanceValidation(driver, password);
  // user should land on homepage after successfully logging in with password
  const homePage = new HomePage(driver);
  await homePage.check_pageIsLoaded();

  // Verify the expected balance on the homepage
  if (ganacheServer) {
    await homePage.check_ganacheBalanceIsDisplayed(ganacheServer);
  } else {
    await homePage.check_expectedBalanceIsDisplayed(expectedBalance); //passing the expectedBalance
  }
};

```
Does it make sense to you? If so, we could do this in a separate PR, and before that, we'll also need to work on disabling the e2e quality gate with a label, because there are around ~40 files that use this method already, and changing them all at once would not be possible with the quality gate.

I can work on this if it make sense to you. However, this shouldn't block this PR, so I'm approving it regardless of this discussion

Reply:
Hi @seaona , thanks for the suggestion, i think it's a good idea ! Agree that we can implement  in a separate PR.